stable_diffusion/stable_diffusion/model_io.py

```diff
@@ -30,6 +30,17 @@
         "diffusion_config": "scheduler/scheduler_config.json",
         "tokenizer_vocab": "tokenizer/vocab.json",
         "tokenizer_merges": "tokenizer/merges.txt",
+    },
+        "nitrosocke/Ghibli-Diffusion": {
```

Comment:
Pre-commit fixed this, though I am unsure if it would not make more sense to remove this model from the list altogether, the selection is very arbitrary right now (with just two entries)

Reply:
Yeah I agree. It would be nice to get a small list that is compatible and just put them there but for now feel free to revert to the original list of 1 model :-) .

Let me know and I 'll go ahead and merge.

Reply:
Some nice next steps ;)

Anyway, back to a single model list now.